README.md

```diff
@@ -1,23 +1,27 @@
-# Merkle Multiproof Inputs Generation For Forge
+> [!WARNING]
+> Note that this library has not had a security review yet, is not gas-efficient, and should only be used for testing.
 
-This is a simple library to generate Merkle Multiproof inputs for OpenZeppelin's `MerkleProof` library. It is written in Solidity and can be used in Forge framework. The library has been tested to work with at most 10k of arbitrary leaves and arbitrary indices of any size.
+# Merkle MultiProof and SingleProof Inputs Generation For Forge
 
-Note that this library is not gas-efficient and should be used for testing purposes only.
+This simple library generates Merkle MultiProof and SingleProof inputs for OpenZeppelin's `MerkleProof` library. It is written in Solidity and can be used in the Forge framework. The library has been tested to work with at most 10k arbitrary leaves and arbitrary indices of any size.
 
 ## Usage
 
-See `MerkleGen.t.sol` for a sample. First, deploy `MerkleGen` contract. Then, prepare the leaves and indices. Finally, call `gen()` to generate the necessary inputs.
+See `MerkleGen.t.sol` for example:
```

Comment:
Let's make this a link to the test file for ease. I believe the syntax is something like
```
[MerkleGen.t.sol](./tests/MerkleGen.t.sol)
```